openpype/settings/entities/schemas/system_schema/schema_general.json

```diff
@@ -138,6 +142,18 @@
                     "multipath": true,
                     "require_restart": true
                 },
+                {
+                    "type": "label",
+                    "label": "Define custom location for artist machine where to unzip versions of Openpype code. By default it is in user app data folder."
+                },
+                {
+                    "type": "path",
+                    "key": "local_openpype_path",
+                    "label": "Custom Local Versions Folder",
+                    "multiplatform": true,
+                    "multipath": true,
```

Comment:
I don't think this should be enabled. If you want to override where versions are stored there should be only one path.

Reply:
oh, I forgot